### PR TITLE
Added terraform scripts for marketplace deployment for Azure, GCP, OCI

### PR DESCRIPTION
**Pull request by SNaveeKumar**
Added terraform scripts for marketplace deployment for Azure, GCP, OCI

### CODE REVIEW

File: azure/1.3.0/fortiaiops.tf [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    create_option     = "FromImage"
+  }
+
+  # Secondary data disks
+  storage_data_disk {
+    name              = "fortiaiopsdatadisk"
+    managed_disk_type = "Standard_LRS"
+    create_option     = "Empty"
+    lun               = 0
+    disk_size_gb      = "500"
```
Comment (mobilesuitzero) [inline azure/1.3.0/fortiaiops.tf]: here log disk size is 500 but is 100 at the other one?  which one is the proper one?

File: azure/1.3.0/fortiaiops.tf [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    create_option     = "FromImage"
+  }
+
+  # Secondary data disks
+  storage_data_disk {
+    name              = "fortiaiopsdatadisk"
+    managed_disk_type = "Standard_LRS"
+    create_option     = "Empty"
+    lun               = 0
+    disk_size_gb      = "100"
```
Comment (mobilesuitzero) [inline azure/1.3.0/fortiaiops.tf]: here is 100, but previous one is 500

File: azure/1.3.0/fortiaiops.tf (hunk 1 above)
Comment (SNaveeKumar) [reply azure/1.3.0/fortiaiops.tf]: As per the team 500 was required, i missed to change maybe.
This was an old commit, where there was a mismatch in the version number, later it was corrected to 2.0.x.

File: azure/1.3.0/fortiaiops.tf (hunk 2 above)
Comment (SNaveeKumar) [reply azure/1.3.0/fortiaiops.tf]: Now the disk size is variablized and fixed to 500

**Review by Joel-Cripps — APPROVED** [review]

**Pull request merged**